doc/install_doc_component.html

```diff
@@ -0,0 +1,197 @@
+<script type="text/javascript">
+var OSName=2;
+if (navigator.appVersion.indexOf("Win")!=-1) OSName=1;
+if (navigator.appVersion.indexOf("Mac")!=-1) OSName=2;
+if (navigator.appVersion.indexOf("X11")!=-1) OSName=3;
+if (navigator.appVersion.indexOf("Linux")!=-1) OSName=3;
+
+$(document).ready(function(){
+    $("ul#tabs li.active").removeClass("active");
+    $("ul#tab li.active").removeClass("active");
+    $("ul#tabs li:nth-child("+OSName+")").addClass("active");
+    $("ul#tab li:nth-child("+OSName+")").addClass("active");
+});
+
+$(document).ready(function(){
+    $("ul#tabs li").click(function(e){
+        if (!$(this).hasClass("active")) {
+            var tabNum = $(this).index();
+            var nthChild = tabNum+1;
+            $("ul#tabs li.active").removeClass("active");
+            $(this).addClass("active");
+            $("ul#tab li.active").removeClass("active");
+            jQuery("ul#tab li:nth-child("+nthChild+")").fadeIn(400).siblings().hide();
+            $("ul#tab li:nth-child("+nthChild+")").addClass("active");
+        }
+    });
+});
+</script>
+
+
+<style type="text/css">
+ul#tabs {
+  list-style-type: none;
+  margin: 0 0 0px 0;
+  padding: 0;
+  text-align: center; }
+  ul#tabs li {
+    font-size:20px;
+    display: inline-block;
+    background-color: #355F7C;
+    border-bottom: solid 5px #C4D8E2;
+    padding: 5px 20px;
+    margin-bottom: 0px;
+    color: black;
+    cursor: pointer; }
```

Comment:
Once you move it to the CSS file, you will need to fix the indentation and style (line return before ending curly brakets) to be consistent with the rest of the file.